README.md

```diff
@@ -4,13 +4,24 @@ Welcome to the FreightHub frontend coding challenge! Please read the following i
 
 **Your goal is to set up an application which enables the user to view and manage shipments.**
 
-## Business need
+# Contents
+
+-   [Business Needs](#business-needs)
+-   [Use cases](#use-cases)
+-   [Evaluation Criteria](#evaluation-criteria)
+    -   [Technology Requirements](#technology-requirements)
+    -   [Code Requirements](https://github.com/freight-hub/freighthub-frontend-challenge/blob/master/Criteria.md#must-have)
+-   [Time limit](#time-limit)
+-   [How to submit?](#how-to-submit)
+-   [Server Side](#how-to-run-api-server)
```

Comment:
```suggestion
-   [How to run the API server](#how-to-run-api-server)
```

Maybe change the name of the script in package.json to "server" instead of "start"